specification/azsadmin/resource-manager/fabric/readme.azsautogen.md

```diff
@@ -0,0 +1,71 @@
+# Fabric Admin
```

Comment:
@RodgeFu  do you know what is readme for? it is right name?

Reply:
Have no idea...

Reply:
@viananth Hi. Do we need this file?

Reply:
This file is needed and we use it for the powershell generation with autorest.  https://github.com/Azure/autorest.powershell